packages/output-handler/test/testMessage.ts

```diff
@@ -0,0 +1,22 @@
+export const testMessage = {
+	Records: [
+		{
+			messageId: '7008c866-eef4-4828-9bd1-48f43744e589',
+			receiptHandle:
+				'AQEBD1XvLLShPITOQt19TXfieYJS+A1i+P3ngKJC5JfyviPESaChjbrM/5BNrq9SphXAQwCt3JBJSmYQszlZHJZpcEUTOZgSWhJuJZRZrQunyCIvgoE2PqaEXDzRKDb97J72ZBdgPk+LJpAzgG/VjLOURNTNxsh56AQy+klkKzzaWO1AYodbWM5NGm8m4kXJRThn1TIGtkKiBPnoOuz7aPlLqj0dUuUiIdjKEfjU/uYOAR4O/4VNYK5NkHdhI+8vliVJ/12AAGA+VRzeibKbBjzF7+lVsYM8tSjxLsuu8oSgSkkgzFMvLLS8F0mq8oiIa2k74+rt2p8JXfk5q+HaB0B9DJMhcAMJ3XnrFc2W2ZzAdFU6Qg9x/eg0MkuDxlTqwYeA6HQ2LE/HLJYX2e7l32MHhLcUHm0Mylmb+vK2TXPf2bE=',
+			body: '{\n  "Type" : "Notification",\n  "MessageId" : "451ba2e2-4507-5933-b7b8-fcdcfdd4b56f",\n  "TopicArn" : "arn:aws:sns:eu-west-1:745349931791:transcription-service-destination-topic-PROD",\n  "Message" : "{\\"id\\":\\"my-first-transcription\\",\\"transcriptionSrt\\":\\"1\\\\n00:00:00,000 --> 00:00:01,300\\\\n This is The Guardian.\\\\n\\\\n2\\\\n00:00:01,300 --> 00:00:06,300\\\\n [Music]\\\\n\\\\n\\",\\"languageCode\\":\\"en\\",\\"userEmail\\":\\"test@test.com\\",\\"originalFilename\\":\\"test.mp3\\"}",\n  "Timestamp" : "2024-02-08T11:10:11.014Z",\n  "SignatureVersion" : "1",\n  "Signature" : "N9Aj8kN72G2OAu5Y5xnZmdGaAjkHEzSCzRRQPMGLtIAPk9Pk1VwjrEc2TQZFSwt4L7Yv7n3XhY7CWexu/nYepbOn9ZG6scTtZ0tGQpsIPBYr7GPjjvvnRYjJtpHeAJzDRgfDAJWnFZhFTxU3XZfy+ZBuLl+86d2WRwe+U3+94AAUJUGr5m1eUGW8U2cj0hl4gPSv1czXwhu+PTyjOqAmWM2bFLWrYnuEIwrdOopBF+gAQGYVkGWbEgy+8GP4O9iMcBXMPyOoCOadNl599vbEX3z68A0kC4I51BSByYAkI8hqr71ONB+DwyYu87E8viu5G8IAq2BIT/jnmHk2fjdjfw==",\n  "SigningCertURL" : "https://sns.eu-west-1.amazonaws.com/SimpleNotificationService-60eadc530605d63b8e62a523676ef735.pem",\n  "UnsubscribeURL" : "https://sns.eu-west-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:eu-west-1:745349931791:transcription-service-destination-topic-PROD:ccd6477c-e3ec-4a0d-aa39-2c49eaa91a59"\n}',
```

Comment:
can you delete anything including the string 745349931791 from this message - I think for local testing we only need `Message`, everything else isn't needed - though you could just set all the values to `test` or something to give an idea of what it looks like on prod